.github/workflows/test-go.yaml

```diff
@@ -76,15 +76,7 @@ jobs:
         run: |
           cd $GITHUB_WORKSPACE/docs/examples/go
           echo "Testing Secure Cell..."
-          enc=$(go run secure_cell_context_imprint.go enc password message context)
-          dec=$(go run secure_cell_context_imprint.go dec password "$enc" context)
-          test "$dec" = "message"
```

Comment:
Here we were checking that encryption/decryption is successful. Sorry, I can’t find where did this check move?

Reply:
> where did this check move?

Nowhere, really. Previous examples were basically a copy of existing integration tests. Now they are replaced with more descriptive example (which is still an automated test of a sort).